src/configuration/app-containers.md

```diff
@@ -462,7 +483,7 @@ Your `.platform.app.yaml` file should include:
 ```yaml
 hooks:
   deploy: |
-    if [ $PLATFORM_BRANCH = "master" ]; then
+    if [ "$PLATFORM_BRANCH" = master ]; then
```

Comment:
Yes (we know exactly what's in the string, so the quotes there are superfluous, whereas we know less about what would be in the PLATFORM_BRANCH variable)